### PR TITLE
doc: simplify composer installation procedure, merge Unit tests related pages, reword/shorten, fix references to old php versions

### PR DESCRIPTION
**Pull request by nodiscc**

### CODE REVIEW

**Comment by nodiscc** [comment]
https://shaarli.readthedocs.io/en/doc-composer/Unit-tests/

File: doc/md/Unit-tests.md [hunk 1]
```diff
@@ -1,126 +1,46 @@
...
-#### Install and enable Xdebug to generate PHPUnit coverage reports
+### Install Xdebug
 
-See http://xdebug.org/docs/install
+Xdebug must be installed and enable for PHPUnit to generate coverage reports. See http://xdebug.org/docs/install.
 
-For Debian-based distros:
 ```bash
+# for Debian-based distributions
 $ aptitude install php5-xdebug
```
Comment (ArthurHoaro) [inline doc/md/Unit-tests.md]:
```suggestion
$ aptitude install php-xdebug
```

Comment (nodiscc) [reply doc/md/Unit-tests.md]: thank you

File: doc/md/Unit-tests.md [hunk 2]
```diff
@@ -1,126 +1,46 @@
...
-$ php composer.phar install
-$ php composer.phar update
-```
-
-#### Install Shaarli dev dependencies
+### Install Shaarli development dependencies
 
 ```bash
 $ cd /path/to/shaarli
+$ composer install
```
Comment (virtualtam) [inline doc/md/Unit-tests.md]: `composer update` should actually be removed as it alters the Composer lock file

File: doc/md/Unit-tests.md [hunk 3]
```diff
@@ -1,126 +1,46 @@
...
-For ArchLinux:
 ```bash
+# for Debian-based distributions
+$ aptitude install php-xdebug
+
+# for ArchLinux:
 $ pacman -S xdebug
 ```
 
 Then add the following line to `/etc/php/php.ini`:
```
Comment (virtualtam) [inline doc/md/Unit-tests.md]:
```suggestion
Then add the following line to `/etc/php/<PHP_VERSION>/cli/php.ini`:
```

**Comment by nodiscc** [comment]
https://shaarli.readthedocs.io/en/doc-composer/Unit-tests/

**Pull request merged**